_haikus/enligth.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Data knowledge
+author: cruz
+---
+
+please serve the content<br>
+data will spread knowledge<br>
```

Comment:
This line needs an extra syllable, maybe something like:

```
this data will spread knowledge <br>
```

Reply:
I really appreciate your help and assistance. I'll change it ASAP.